### ISSUE TITLE
2024-04-23-10

### ISSUE THREAD

**Issue by Mr-wang0129** [Mr-wang0129/ChatDailyPapers#3]
# image matching
## Scene Coordinate Reconstruction: Posing of Image Collections via Incremental Learning of a Relocalizer
1. **Title:** 场景坐标重建：通过增量学习重定位图像集合

2. **Authors:** Eric Brachmann, Jamie Wynn, Shuai Chen, Tommaso Cavallari, Áron Monszpart, Daniyar Turmukhambetov, and Victor Adrian Prisacariu

3. **Affiliation:** Niantic

4. **Keywords:** scene coordinate reconstruction, incremental learning, visual relocalization, structure-from-motion, neural scene representations

5. **Urls:** [Paper](http://arxiv.org/abs/2404.14351v1), [Project page](https://nianticlabs.github.io/acezero/), Github: None

6. **Summary:** 

    - (1): 这篇文章的研究背景是计算机视觉中的图像集合中的相机参数估计问题。
 
    - (2): 以往的方法主要是基于特征匹配的结构运动（Structure-from-Motion, SfM）工具，其通过增量重建来解决估计相机参数的任务。但这些方法存在一些问题，包括需要粗略的初始姿势、对姿势分布的先验知识或大致顺序的输入。而文章提出的方法则受到增量SfM以及视觉重定位的启发，采用学习场景坐标回归的方法，能够快速、有效地进行重定位，同时不需要姿势先验或顺序输入，可以处理数千张图像。

    - (3): 文章提出了一种学习的重定位方法，将增量SfM重新解释为对视觉重定位的迭代应用和改进。通过学习场景坐标回归，构建了隐式的神经场景表示，使得可以从未定位的图像中估计相机姿势。

    - (4): 文章的方法在相机姿势估计方面取得了与基于特征匹配的SfM相当的精度，通过新颖的视图合成进行了验证。这种性能支持了他们的研究目标。

    - (5): 本文的贡献在于提出了一种新颖的学习场景坐标回归方法，实现了对图像集合中相机参数的准确估计，不受传统方法的限制。

- **Abstrat**: We address the task of estimating camera parameters from a set of images depicting a scene. Popular feature-based structure-from-motion (SfM) tools solve this task by incremental reconstruction: they repeat triangulation of sparse 3D points and registration of more camera views to the sparse point cloud. We re-interpret incremental structure-from-motion as an iterated application and refinement of a visual relocalizer, that is, of a method that registers new views to the current state of the reconstruction. This perspective allows us to investigate alternative visual relocalizers that are not rooted in local feature matching. We show that scene coordinate regression, a learning-based relocalization approach, allows us to build implicit, neural scene representations from unposed images. Different from other learning-based reconstruction methods, we do not require pose priors nor sequential inputs, and we optimize efficiently over thousands of images. Our method, ACE0 (ACE Zero), estimates camera poses to an accuracy comparable to feature-based SfM, as demonstrated by novel view synthesis. Project page: https://nianticlabs.github.io/acezero/

## Heterogeneous Face Recognition Using Domain Invariant Units
1. **Title:** Heterogeneous Face Recognition Using Domain Invariant Units (使用域不变单元进行异构人脸识别)

2. **Authors:** Anjith George and Sébastien Marcel

3. **Affiliation:** Idiap Research Institute, Martigny, Switzerland (瑞士马尔蒂尼伊迪阿普研究所)

4. **Keywords:** Face Recognition, Heterogeneous Face Recognition, Domain Invariant Units, Biometrics

5. **Urls:** [Paper](http://arxiv.org/abs/2404.14343v1), [Github:None]

6. **Summary:** 

    - (1):该文章研究背景为人脸识别系统在异构环境中的应用挑战，旨在解决不同领域之间的人脸图像匹配问题。
  
    - (2):过去的方法包括特征提取方法如DoG和SIFT以及通用空间投影方法，但面临领域差异和数据匮乏等问题。文章的方法有充分的动机。
  
    - (3):本文提出了一种新的方法，通过预训练的人脸识别模型作为教师网络，在对抗蒸馏框架中学习域不变网络层（DIU），从而减小领域差距。该方法能有效地使用有限的配对训练数据进行训练。
  
    - (4):该方法在多个基准测试上进行了广泛评估，在性能上优于现有方法。该方法的性能支持其目标，即增强预训练模型，使其更适应数据变化的范围。
  
    - (5):本文的贡献包括提出了一种基于教师-学生蒸馏框架的异构人脸识别方法，该方法利用少量配对数据可以改善预训练模型的适应性，并在多个标准基准上进行了有效性验证。

- **Abstrat**: Heterogeneous Face Recognition (HFR) aims to expand the applicability of Face Recognition (FR) systems to challenging scenarios, enabling the matching of face images across different domains, such as matching thermal images to visible spectra. However, the development of HFR systems is challenging because of the significant domain gap between modalities and the lack of availability of large-scale paired multi-channel data. In this work, we leverage a pretrained face recognition model as a teacher network to learn domaininvariant network layers called Domain-Invariant Units (DIU) to reduce the domain gap. The proposed DIU can be trained effectively even with a limited amount of paired training data, in a contrastive distillation framework. This proposed approach has the potential to enhance pretrained models, making them more adaptable to a wider range of variations in data. We extensively evaluate our approach on multiple challenging benchmarks, demonstrating superior performance compared to state-of-the-art methods.

## From Modalities to Styles: Rethinking the Domain Gap in Heterogeneous Face Recognition
1. **Title:** 从模态到风格：重新思考异构人脸识别中的域差距

2. **Authors:** Anjith George, S´ebastien Marcel

3. **Affiliation:** Idiap Research Institute

4. **Keywords:** Face Recognition, Heterogeneous Face Recognition, Style transfer, Instance Normalization, Biometrics

5. **Urls:** [Paper](http://arxiv.org/abs/2404.14247v1), Github: None

6. **Summary:** 

   - (1): 该论文研究的背景是异构人脸识别（HFR）领域的跨域匹配问题。
  
   - (2): 过去的方法主要是在同质域内进行匹配，但在异构设置中匹配更有利。然而，这种跨域匹配面临域差距的挑战，且大部分目标HFR模态的数据集有限。本文的方法有很好的动机，将不同模态视为不同的风格，并提出一种调制目标模态特征图的方法来解决域差距问题。
  
   - (3): 本文提出了一种新的条件自适应实例调制（CAIM）模块，可以无缝地嵌入现有的FR网络中，将其转换为HFR准备系统。CAIM模块调制中间特征图，有效地适应源模态的风格，并弥合域差距。该方法允许使用一小组配对样本进行端到端训练。
  
   - (4): 该方法在各种具有挑战性的HFR基准测试上进行了广泛评估，表明优于现有的方法。性能支持了他们的目标。
  
   - (5): 该论文的贡献在于提出了一种有效的方法来解决异构人脸识别中的域差距问题，从而提高了系统的性能和适用性。

- **Abstrat**: Heterogeneous Face Recognition (HFR) focuses on matching faces from different domains, for instance, thermal to visible images, making Face Recognition (FR) systems more versatile for challenging scenarios. However, the domain gap between these domains and the limited large-scale datasets in the target HFR modalities make it challenging to develop robust HFR models from scratch. In our work, we view different modalities as distinct styles and propose a method to modulate feature maps of the target modality to address the domain gap. We present a new Conditional Adaptive Instance Modulation (CAIM ) module that seamlessly fits into existing FR networks, turning them into HFR-ready systems. The CAIM block modulates intermediate feature maps, efficiently adapting to the style of the source modality and bridging the domain gap. Our method enables end-to-end training using a small set of paired samples. We extensively evaluate the proposed approach on various challenging HFR benchmarks, showing that it outperforms state-of-the-art methods. The source code and protocols for reproducing the findings will be made publicly available

# homography estimation
